cyclonedx/model/bom.py

```diff
@@ -263,7 +287,107 @@ def has_component(self, component: Component) -> bool:
         Returns:
             `bool` - `True` if the supplied Component is part of this Bom, `False` otherwise.
         """
-        return component in self._components
+        if not self.components:
+            return False
+        return component in self.components
+
+    @property
+    def services(self) -> Optional[List[Service]]:
+        """
+        Get all the Services currently in this Bom.
+
+        Returns:
+             List of `Service` in this Bom or `None`
+        """
+        return self._services
+
+    @services.setter
+    def services(self, services: Optional[List[Service]]) -> None:
+        self._services = services
+
+    def add_service(self, service: Service) -> None:
```

Comment:
this would not be needed, if `self._services` was a list at all times and would be returned by the setter.
then the users could run `myBom.services.append()`